src/LiveSplit.SpeedGuidesLive/src/SGLGuideWindow.cs

```diff
@@ -181,50 +185,81 @@ private void ValidateComponents(bool force)
             }
         }
 
-        private void ClearLabels()
+        /// <summary>
+        /// Convert markdown to a valid html page.
+        /// </summary>
+        /// <param name="text">Markdown string that is in the process of being rendered.</param>
+        /// <returns>A string containing valid html that can be placed into a web browser.</returns>
+        private string GenerateHtmlFromMD(string text)
         {
-            foreach(GrowLabel label in m_labels)
+            // set the styles for the browser window based on user settings
+            string html = "<html><head><style>";
+            html += "html,body{";
+            html += "background-color: rgb(" + m_backgroundColor.R.ToString() + ", " + m_backgroundColor.G.ToString() + ", " + m_backgroundColor.B.ToString() + ");";
+            html += "color: rgb(" + m_textColor.R.ToString() + ", " + m_textColor.G.ToString() + ", " + m_textColor.B.ToString() + ");";
+            html += "font-family: " + m_component.Settings.GuideFont.Name + ";";
+            html += "font-size: " + m_component.Settings.GuideFont.Size.ToString() + "px;";
+            html += "}";
+            html += "img{max-width:100%;}";
+            html += "pre{word-wrap:break-word;}";
+            html += "</style></head><body>";
```

Comment:
I'd recommend using a  single multiline string interpolation rather than concatenating 25 or so times. It would also be much more reasable and remove all the calls to `.ToString()`